metadata-ws/src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/services/TaxonomyTreeService.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.ampt2d.metadata.persistence.services;
+
+import java.util.List;
+
+public interface TaxonomyTreeService {
+
+    List<Long> findAllSpeciesAndSubspeciesTaxonomyIdsInATaxonomyTreeByTaxonomyId(long taxonomyId);
+
+    List<Long> findAllSpeciesAndSubspeciesTaxonomyIdsInATaxonomyTreeByTaxonomyName(String taxonomyName);
```

Comment:
Again, why do we still have `TaxonomyTreeService` if we decided to get rid of TaxonomyTree entirely?

Reply:
Yes, but since we do not have a `TaxonomyTree`, surely we should not have such thing as a `TaxonomyTreeService`; at least call it `TaxonomyService`